desktop/components/search_bar/view.coffee

```diff
@@ -169,15 +169,19 @@ module.exports = class SearchBarView extends Backbone.View
   emptyItemClick: ->
     analyticsHooks.trigger 'search:empty-item:click',
       query: @query
-      label: modelNameAndIdToLabel 'user-query', 'query'
+      item_number: 1
+      item_type: 'search'
+      destination_path: "#{sd.APP_URL}/search?q=#{@query}"
     @selected = true
     location.assign "/search?q=#{@query}"
 
   selectResult: (e, model) ->
     return @emptyItemClick() unless model
     analyticsHooks.trigger 'search:item:click',
       query: @query
-      label: modelNameAndIdToLabel model.get('display_model'), model.id
+      item_number: model.collection.models.findIndex( (result) -> return result.id == model.id ) + 1
```

Comment:
I thought this was kind of a contrived way of getting the [item number](https://github.com/artsy/collector-experience/issues/251#issuecomment-300321049). Let me know if you have a better solution proposed.